shell/shared/renderSession/RenderSessionConfig.h

```diff
@@ -16,7 +16,6 @@ struct RenderSessionConfig {
   std::string displayName;
   BackendVersion backendVersion;
   igl::TextureFormat swapchainColorTextureFormat = igl::TextureFormat::BGRA_UNorm8;
-  igl::TextureFormat depthTextureFormat = igl::TextureFormat::Z_UNorm16;
```

Comment:
There's a lot of our internal code that relies on this field. Is it possible to add stencil buffer support without getting rid of it?

Reply:
The field depthTextureFormat  is add by me in this commit.
 https://github.com/facebook/igl/commit/49d674c4019c3e8d107cfbaaa04bd4f926c83c06

This PR only revert the commit before, add move the depthTextureFormat field into RenderSession.

Reply:
> The field depthTextureFormat is add by me in this commit.[49d674c](https://github.com/facebook/igl/commit/49d674c4019c3e8d107cfbaaa04bd4f926c83c06)
> 

I looked into it, the tests do not pass because the file `shell/ios/ViewController.mm` needs to be updated as well. It still uses `depthTextureFormat`.

Reply:
I have fixed iOS compile error.

The complete adaptation of the iOS stencil buffer is another PR(https://github.com/facebook/igl/pull/221).
 It should be merged with the latest main branch and reviewed after this PR is approved.